src/main/flight/mixer.c

```diff
@@ -759,6 +760,8 @@ void mixTable(void)
             axisPID[PITCH] * currentMixer[i].pitch +
             axisPID[ROLL] * currentMixer[i].roll +
             -mixerConfig->yaw_motor_direction * axisPID[YAW] * currentMixer[i].yaw;
+
+        if (batteryConfig->vbatPidCompensation) motor[i] *= calculateVbatPidCompensation();  // Add voltage compensation
```

Comment:
Would it be better to calculate it once [`float vbatCompensation = calculateVbatPidCompensation()`], then apply it over and over again?

Also - there's generally a flat spot / deadzone at the lower end of the motor, so for more precision, you should use the form:  `motor[i] = throttle_zero + (motor[i]-throttle_zero)*vbatCompensation`. I estimated 'throttle_zero' to be somewhere between minCommand and minThrottle (since the props are typically spinning at minThrottle). For SimonK, the bottom 1/16 (6.25%) is 0, I'm not sure about blheli. This effect is much less significant than the 20% change in battery voltage, so perhaps negligible.